src/bpmat/KSM.cpp

```diff
@@ -1395,6 +1404,8 @@ int GCROT::solve(TACSVec *b, TACSVec *x, int zero_guess) {
     // mat->mult( x, R );      // R = A*x
     // R->axpy( -1.0, b );     // R = A*x - b
     // R->scale( -1.0 );
+    iterCount += niters;
+    resNorm = R->norm();
```

Comment:
No need to compute this norm outside the solve flag right?

Reply:
Fixed